server/backend/database/mongo/client.go

```diff
@@ -268,6 +268,28 @@ func (c *Client) listProjectInfos(
 		return nil, fmt.Errorf("fetch project infos: %w", err)
 	}
 
+	if len(infos) < pageSize {
+		opts.SetLimit(int64(pageSize - len(infos)))
+		if err != nil {
```

Comment:
Is there any reason for this error check?

Reply:
It's useless error checking. I removed it.